client/src/main/scala/drt/client/actions/Actions.scala

```diff
@@ -29,9 +29,11 @@ object Actions {
 
   case class RunAllSimulations() extends Action
 
+  case class RunTerminalSimulations(terminalName: TerminalName) extends Action
+
   case class RunSimulation(terminalName: TerminalName, queueName: QueueName, desks: List[Int]) extends Action
 
-  case class SetFixedPoints(fixedPoints: String) extends Action
+  case class SetFixedPoints(fixedPoints: String, terminalName: Option[String]) extends Action
```

Comment:
We have two use cases, one where fixed points are updated on the terminal page, and one where they're updated from the server. Thus an optional terminalName. Guess we could make it mandatory and call it for each separate terminal when we get them back from the server